src/mmw/js/src/core/templates/shareModal.html

```diff
@@ -1,19 +1,28 @@
 <div class="modal-dialog modal-dialog-sm">
     <div class="modal-content">
         <div class="modal-header">
-            <h2>Share Project</h2>
+            <h2>Share {{ text }}</h2>
         </div>
         <div class="modal-body">
-            <div class="custom-input-group">
-              <input type="text" required value="{{ url }}">
-              <span class="highlight"></span>
-              <span class="bar"></span>
-              <label>URL</label>
-            </div>
+            {% if guest %}
+                <p>Sorry, only saved {{ text }}s are able to be shared.  Please log in to share your work.</p>
```

Comment:
Verified that this works, though we should make a note an issue that we should prompt a save before sharing once they are logged in.